codegen/src/api/mod.rs

```diff
@@ -169,23 +169,23 @@ impl RuntimeGenerator {
             ),
             (
                 "sp_core::crypto::AccountId32",
-                parse_quote!(#crate_path::ext::sp_core::crypto::AccountId32),
+                parse_quote!(#crate_path::utils::account_id::AccountId32),
```

Comment:
Better if `AccountId32` is reexported at `utils`, robust to refactoring of namespace.